commands.js

```diff
@@ -1500,6 +1502,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was enabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-green\"><b>The ladder is now back.</b><br />Rated games will update the ladder now.</div>");
 	},
+	enableladderhelp: ["Enables the ladder, which will allow rated battles to commence and allows the ladder to update. Requires: ~"],
```

Comment:
Here too: "enableladderhelp: ["*/enableladder* - Enables the ladder, which will allow rated battles to commence and allows the ladder to update. Requires: ~"],"